### ISSUE TITLE
Building failure should give better error message 

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/pak#67]
This is not much help:

```
ℹ Building GO.db 3.6.0
✖ Failed to build GO.db 3.6.0 5.3s
Error: Failed to build source package 'GO.db'.
```

**Comment by gaborcsardi** [comment]
This would ideally need https://github.com/r-lib/pkgbuild/pull/78 so the error message and output from pkgbuild is nice.

**Comment by jimhester** [comment]
Closed by #182.
